samcli/hook_packages/terraform/copy_terraform_built_artifacts.py

```diff
@@ -217,27 +217,24 @@ def cli_exit():
     argparser.add_argument(
         "--terraform-project-root",
         type=str,
-        required=False,
+        required=True,
```

Comment:
If this script is moved, the corresponding pyinstaller hook also needs to be changed to ensure that this file is included.

Reply:
It seems like it already includes the samcli, so I believe I only need to remove these files now? https://github.com/sriram-mv/aws-sam-cli/blob/85cfa00ecb49e12ae04685b74f20617656e0a917/installer/pyinstaller/hook-samcli.py#L8-L10

Reply:
https://github.com/sriram-mv/aws-sam-cli/blob/85cfa00ecb49e12ae04685b74f20617656e0a917/installer/pyinstaller/hook-samcli.py#L10 

`include_datas` is the piece that matters there, it collects all the py files first and then only includes the files that are passed to `include_datas`, and that takes in a path.

Reply:
got it, thank you! 
done